### PR TITLE
Create get_dimensions.md

### PR DESCRIPTION
**Pull request by gwww**
Missing document.

Was not sure how to populate "Since" from the git hash. Here is the commit: https://github.com/wez/wezterm/commit/ba5d50ba9e7c1c9ff03f7f4c0c7f7562f08e7134

### CODE REVIEW

File: docs/config/lua/window/get_dimensions.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# `window:get_dimensions()`
+
+*Since: 20210307*
```
Comment (wez) [inline docs/config/lua/window/get_dimensions.md:3]:
```suggestion
*Since: 20210314-114017-04b7cedd*
```

**Comment by wez** [comment]
Thanks for this!  Would you mind updating the since to match the suggested change?  That'll make it convenient for me to merge!

**Comment by gwww** [comment]
How did you compute the since?

**Comment by wez** [comment]
I looked at https://wezfurlong.org/wezterm/changelog.html to find the first release with a date later than 20210307

**Pull request merged**